CHANGELOG.md

```diff
@@ -99,7 +99,9 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
   well as the error message)
   ([issue](https://github.com/hashicorp/boundary/issues/1305),
   [PR](https://github.com/hashicorp/boundary/pull/1384))
-
+* server: Fixed a regression in 0.4.0 that caused the controller to not shut
```

Comment:
It's actually not a regression...this was changed post 0.4.0 so this is just a bug fix for the panic!

Reply:
Oh! I think I misread our offline convo then... will fix!